request-authentication.md

```diff
@@ -6,13 +6,85 @@ pagination_next: null
 pagination_prev: null
 ---
 
+import ApiSchema from '@theme/ApiSchema';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 # Request authentication
 
 A notification request from webhooks can be verified by using the `host`, `x-ms-date`, `x-ms-content-sha256`, and `authorization` headers with the
 unique secret received when the webhook was registered. The authorization is a
 sha256 HMAC hash of the request date, content, and URI that is created using the
 webhook secret.
 
+<Tabs
+    defaultValue="js" 
+    groupId="example-choice" 
+    values={[ {label: 'JavaScript', value: 'js'}, {label: '.Net C#', value: 'csharp'}, ]}>
+
+<TabItem value="js">
+
+To run: `node webhook.mjs`
+```js
+import crypto from "crypto";
+
+// The secret is found in the response body when you register a webhook. 
+// Note: The secret is unique per hook registration (EventType+YourURL).
+const secret = 'A0+AeKBRG2KRGvnNwJpQlb6IJFk48CKXCIcrLoHncVJKDILsQSxS6NWCccwWm6r6FhGKhiHTBsG2wo/xU6FY/A=='
+
+// The complete url of the webhook, including query parameters if applicable. I.e. https://yoursite.com/webhooks/vmp/v1
```

Comment:
I think this comment can be removed, it should be pretty self explanatory in my opinion.
An url is an url, it's not the host name or something like that.

Maybe change variable name `requestUrl` to `callbackUrl = <your company url>`?

Reply:
Removed comment in [9287d3c](https://github.com/vippsas/webhooks-api/pull/23/commits/9287d3c361c4f0e6e49dd5cfd37f6b5853c4c9ca)